examples/run_hid_host.py

```diff
@@ -0,0 +1,465 @@
+    )
+
+    if (len(service_record_handles) < 1):
+        print(color('BT HID Device service not found on peer device!!!!','red'))
+        await sdp_client.disconnect()
+        return
+
+    # For BT_HUMAN_INTERFACE_DEVICE_SERVICE service, get all its attributes
+    for service_record_handle in service_record_handles:
+        attributes = await sdp_client.get_attributes(
+            service_record_handle, [SDP_ALL_ATTRIBUTES_RANGE]
+        )
+        print(
+            color(f'SERVICE {service_record_handle:04X} attributes:', 'yellow')
+        )
+        print(color(f'SDP attributes for HID device','magenta'))
+        for attribute in attributes:
+            if  attribute.id == SDP_SERVICE_RECORD_HANDLE_ATTRIBUTE_ID :
+                print(color('  Service Record Handle : ', 'cyan'), hex(attribute.value.value))
+
+            elif  attribute.id == SDP_SERVICE_CLASS_ID_LIST_ATTRIBUTE_ID :
+                print(color('  Service Class : ', 'cyan'), attribute.value.value[0].value)
+
+            elif  attribute.id == SDP_BROWSE_GROUP_LIST_ATTRIBUTE_ID :
+                print(color('  SDP Browse Group List : ', 'cyan'), attribute.value.value[0].value)
+
+            elif  attribute.id == SDP_PROTOCOL_DESCRIPTOR_LIST_ATTRIBUTE_ID :
+                print(color('  BT_L2CAP_PROTOCOL_ID : ', 'cyan'), attribute.value.value[0].value[0].value)
+                print(color('  PSM for Bluetooth HID Control channel : ', 'cyan'), hex(attribute.value.value[0].value[1].value))
+                print(color('  BT_HIDP_PROTOCOL_ID : ', 'cyan'), attribute.value.value[1].value[0].value)
+
+            elif  attribute.id == SDP_LANGUAGE_BASE_ATTRIBUTE_ID_LIST_ATTRIBUTE_ID :
+                print(color('  Lanugage : ', 'cyan'), hex(attribute.value.value[0].value))
+                print(color('  Encoding : ', 'cyan'), hex(attribute.value.value[1].value))
+                print(color('  PrimaryLanguageBaseID : ', 'cyan'), hex(attribute.value.value[2].value))
+
+            elif  attribute.id == SDP_BLUETOOTH_PROFILE_DESCRIPTOR_LIST_ATTRIBUTE_ID :
+                print(color('  BT_HUMAN_INTERFACE_DEVICE_SERVICE ', 'cyan'), attribute.value.value[0].value[0].value)
+                print(color('  HID Profileversion number : ', 'cyan'), hex(attribute.value.value[0].value[1].value))
+
+            elif  attribute.id == SDP_ADDITIONAL_PROTOCOL_DESCRIPTOR_LIST_ATTRIBUTE_ID :
+                print(color('  BT_L2CAP_PROTOCOL_ID : ', 'cyan'), attribute.value.value[0].value[0].value[0].value)
+                print(color('  PSM for Bluetooth HID Interrupt channel : ', 'cyan'), hex(attribute.value.value[0].value[0].value[1].value))
+                print(color('  BT_HIDP_PROTOCOL_ID : ', 'cyan'), attribute.value.value[0].value[1].value[0].value)
+
+            elif  attribute.id == SDP_HID_SERVICE_NAME_ATTRIBUTE_ID :
+                print(color('  Service Name: ', 'cyan'), attribute.value.value)
+                HID_Service_Name = attribute.value.value
```

Comment:
all these `HID_Abc_Def` variables are set here but don't seem to be used anywhere. What is their role?

Reply:
SDP HID related attributes were maintained for future use. But since they are not currently used by application, removed the variables and we are just printing them.